proto/src/determined/api/v1/trial.proto

```diff
@@ -481,6 +481,8 @@ message AckAllocationPreemptionSignalRequest {
   };
   // The allocation that is acknowledging the request.
   string allocation_id = 1;
+  // The resource manager for the request.
+  string resource_manager = 2;
```

Comment:
I don't think these need RM since they get routed to the allocation service?

Reply:
stray change, sorry